src/form.js

```diff
@@ -396,7 +393,7 @@ const mapState = state => {
 		fallbackTicketPathToShow: getFallbackTicketPathToShow( state ),
 		fallbackTicketResponse: getFallbackTicketResponse( state ),
 		fallbackTicketStatus: getFallbackTicketStatus( state ),
-		getAuth: getHappychatAuth( state ),
+		happychatAuthentication: authenticator.authorizeChat( state ),
```

Comment:
We may want to rename this to property to `authentication`.

I've simplified names and removed the `happychat` prefix/suffix wherever possible. It might make sense to have that when Happychat was within Calypso, but that's no longer true! We may still have some names with those prefixes that need refactor, but for new names I'd drop them.

Reply:
Great suggestion, updated.